global.json

```diff
@@ -22,8 +22,7 @@
       "version": "16.8",
       "components": [
         "Microsoft.VisualStudio.Component.VC.ATL",
-        "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
-        "Microsoft.VisualStudio.Component.Windows10SDK.17134"
+        "Microsoft.VisualStudio.Component.VC.Tools.x86.x64"
```

Comment:
I ran into this the other day - we are going to need *some* windows SDK I think, but that one no longer seems to be available in VS2022.

Reply:
Yeah, @dougbu said he'll follow up with the Arcade folks

Reply:
Just confirmed `Microsoft.VisualStudio.Component.Windows10SDK.18362` (the new minimum in 2022) is available in 2019 too. I'm asking in the Arcade channel for that component to be added to the VS configuration on all agents in their next roll-out or perhaps the week after

Reply:
/fyi https://teams.microsoft.com/l/message/19:1dad2081c8634f34915d88dce6220265@thread.skype/1626396237512?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=4d73664c-9f2f-450d-82a5-c2f02756606d&parentMessageId=1626396237512&teamName=.NET%20Core%20Eng%20Services%20Partners&channelName=Arcade&createdTime=1626396237512